Sources/Support/StoreKitMessagesHelper.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  StoreKitMessagesHelper.swift
+//
+//  Created by Antonio Rico Diez on 27/9/23.
+
+import StoreKit
+
+class StoreKitMessagesHelper {
+
+    private let systemInfo: SystemInfo
+    private let showStoreKitMessagesAutomatically: Bool
+
+    // TODO: Need to improve this :/
+    private var deferredMessages: [Any] = []
```

Comment:
Should this type be `Message` instead of `Any`?

Reply:
+1

Reply:
Yeah, that's why I have the `TODO`. Since `Message` is iOS 16.0+, I need to make this whole class `@available` that, which was a bit annoying since I needed to pass that around. On the other hand, I could make this like that and make it nullable in the `PurchasesOrchestrator`, which is probably the best way to go for this.